action.yml

```diff
@@ -14,130 +14,13 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    # let the container take ownership of the repo dir, in case the user wants to check in the results
-    # workaround to https://github.com/actions/runner/issues/2033
-    - run: chown -R $(id -u):$(id -g) $PWD
```

Comment:
We could keep this, but I think it's better to just run the docker container with matching uid/gid from the vm. This could be simpler if we used docker_run and just killed action.yml.

The repo wishing to build would just have:
```yaml
name: CI
on:
  [ pull_request ]
jobs:
  render:
    runs-on: ubuntu-latest

    steps:
      - name: Check out repository
        uses: actions/checkout@v2
      - name: Launch Action
      - name: build pdf
        run: docker_run --pdf=output.pdf --docx=output.docx --latex=output.tex input.md 
```
docker_run already sets uid/gid. I'd like to kill action.yml :-p

Reply:
the default ci images are setup with docker already too.